wireless_modules/middle_module/main.py

```
@@ -0,0 +1,89 @@
+# Note: You can remove all the PRINT statements when everything is already tested
+
+import machine
+import ubinascii
+import utime
+
+# To get access to the 'MQTT_BROKER' variable in config.py
+try:
+    import config
+except FileNotFoundError:
+    print('Error importing config.py, ensure a local version of config.py exists')
+
+# To get access to the Client class in the MQTT_Client_class file
+try:
+    from MQTT_Client_class import Client
+except:
+    print("Error importing MQTT_Client_class.py, ensure a local version of the file exists")
```

`MQTT_Client_class.py`should always exist unless intentionally deleted, nevertheless I don't think we need an explicit try-except for this case, better to just throw the error rather than keep running without the import (which will cause a crash anyway)